### PR TITLE
Restoring Files

### PR DESCRIPTION
**Pull request by faditawfik21**
This a PR to restored the two deleted files on the main PR.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3840,35 +16948,36 @@
             }
         },
         "extract-zip": {
-            "version": "2.0.1",
-            "resolved": "https://registry.npmjs.org/extract-zip/-/extract-zip-2.0.1.tgz",
-            "integrity": "sha512-GDhU9ntwuKyGXdZBUgTIe+vXnWj0fppUEtMDL0+idd5Sta8TGpHssn/eusA9mrPr9qNDym6SxAYZjNvCn/9RBg==",
-            "dev": true,
+            "version": "1.7.0",
```
Comment (refortie) [inline package-lock.json]: This is the problem with this file. We need the 2.0.1 version, but I don't think we should manually updating this file

Comment (faditawfik21) [reply package-lock.json]: I realized I used the one from main but just updated it with our internship master one.

Comment (nertim) [reply package-lock.json]: I think my main complain with the other PR was the tasks.json file.. this one I was okay with.

Comment (faditawfik21) [reply package-lock.json]: Sounds good, I restored the `package-lock.json` file and now the only change in this PR is re-adding the `tasks.json` file.

**Pull request closed without merging**